.circleci/config.yml

```diff
@@ -377,6 +377,29 @@ jobs:
       - save_cache: *save-terser-cache
       - save_cache: *save-babel-client-cache
       - store-artifacts-and-test-results
+      - run:
+          name: Notify ICFY of failed build
+          when: on_fail
+          command: |
+            #
+            # This block should not cause a test failure and block PRs.
+            # The shell should never error and exit 0 to indicate success.
+            #
+            set +o errexit
+            ANCESTOR_SHA1=$(git merge-base HEAD origin/master)
+            curl                                                                      \
+              -X POST                                                                 \
+              "http://api.iscalypsofastyet.com:5000/failed-build?secret=$ICFY_SECRET" \
```

Comment:
I added and deployed a `/stats-submit-failed` endpoint (naming consistent with the `/stats-submit` endpoint used for success) in https://github.com/Automattic/icfy/commit/cab6e729b590c44ad83524ba072d7e82b2673114. Please update the URL 🙏

Reply:
It's apparently `submit-stats-failed` 😉